test/common/trafficmanager/validator/backend.go

```diff
@@ -67,3 +67,14 @@ func IsTrafficManagerBackendDeleted(ctx context.Context, k8sClient client.Client
 		return nil
 	}, timeout, interval).Should(gomega.Succeed(), "Failed to remove trafficManagerBackend %s ", name)
 }
+
+// ConsistentlyTrafficManagerConsistentlyExist validates whether the backend consistently exists.
+func ConsistentlyTrafficManagerConsistentlyExist(ctx context.Context, k8sClient client.Client, name types.NamespacedName) {
```

Comment:
`ConsistentlyTrafficManagerConsistentlyExist` seems weird? Are we following any naming convention?